Adyen/UI/Form/Items/Address/FormFullAddressItem.swift

```diff
@@ -0,0 +1,149 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Abstract class for form item to show address form.
+/// :nodoc:
+open class AnyAddressItem: FormValueItem<AddressInfo, AddressStyle> {
+
+    /// :nodoc:
+    public private(set) var subitems: [FormItem] = []
+
+}
```

Comment:
this is not needed, since every `FormItem` now has `var subitems: [FormItem]`

Reply:
indeed, `AnyAddressItem` doesn't make much sense any more 🤔